src/Tasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -670,8 +671,24 @@ private static void SignPEFile(X509Certificate2 cert, Uri timestampUrl, string p
                         // everything was fine
                         break;
                     case 1:
-                        // error, report it
-                        throw new ApplicationException(String.Format(CultureInfo.InvariantCulture, resources.GetString("SecurityUtil.SigntoolFail"), path, signTool.StandardError.ReadToEnd()));
+                        {
+                            // error, retry with /t if timestamp url was given
```

Comment:
Another possible way to accomplish the same:
(1) Change the name of SignPEFile to e.g., SignPEFileInternal, add an extra parameter "bool useRFC3161Timestamp", and returns true if it is successful.
(2) Then the new SignPEFile becomes something like:
```
private static void SignPEFile(X509Certificate2 cert, Uri timestampUrl, string path, System.Resources.ResourceManager resources, bool useSha256)
{
    if (!SignPEFileInternal(cert, timestampUrl, path, resources, useSha256, useRFC3161Timestamp: true))
    {
        SignPEFileInternal(cert, timestampUrl, path, resources, useSha256, useRFC3161Timestamp: false);
    }
}
```